.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies and run tests
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
I believe 3.6 is going to be unsupported so maybe don't include it?

Reply:
sounds reasonable, but looks like 3.6 is still used - i.e. AzureML compute nodes are still having 3.6 by default as far as I know. Let's keep it for some time.